mooncake-store/include/rpc_service.h

```diff
@@ -255,6 +259,29 @@ class WrappedMasterService {
         return response;
     }
 
+    RemoveAllResponse RemoveAll() {
+        ScopedVLogTimer timer(1, "RemoveAll");
+        timer.LogRequest("action=remove_all_objects");
+
+        // Increment request metric
+        MasterMetricManager::instance().inc_remove_all_requests();
+
+        RemoveAllResponse response;
+        const long removed_count = master_service_.RemoveAll();
+
+        // Track failures if needed
+        if (removed_count < 0) {
```

Comment:
The removed_count should theoretically never be less than 0, so we might consider using an assert directly.